src/main/java/com/whiletrue/tododemo/service/impl/TaskServiceImpl.java

```diff
@@ -16,29 +20,31 @@
 public class TaskServiceImpl implements TaskService {
 
     private final TaskRepository taskRepository;
+    private final UserRepository userRepository;
 
-    public TaskServiceImpl(TaskRepository taskRepository) {
+    public TaskServiceImpl(TaskRepository taskRepository,
```

Comment:
I think you are missing `Autowired` annotation here.

Reply:
It is working also without @Autowired, because I setup the repository through constructor